Examples/LJ_langevin.py

```diff
@@ -19,18 +21,29 @@
     lj_fluid.topology, sigma=sigma, epsilon=epsilon, cutoff=cutoff
 )
 
+
 from chiron.states import SamplerState, ThermodynamicState
 
+PRNG.set_seed(1234)
 # define the sampler state
 sampler_state = SamplerState(
-    x0=lj_fluid.positions, box_vectors=lj_fluid.system.getDefaultPeriodicBoxVectors()
+    x0=lj_fluid.positions,
+    current_PRNG_key=PRNG.get_random_key(),
+    box_vectors=lj_fluid.system.getDefaultPeriodicBoxVectors(),
+)
+
+velocities = initialize_velocities(
+    300 * unit.kelvin, lj_fluid.topology, PRNG.get_random_key()
 )
 
+
 # define the thermodynamic state
 thermodynamic_state = ThermodynamicState(
-    potential=lj_potential, temperature=300 * unit.kelvin
+    potential=lj_potential,
+    temperature=300 * unit.kelvin,
 )
 
+
 from chiron.neighbors import NeighborListNsqrd, OrthogonalPeriodicSpace
```

Comment:
`NeighborListNsqrd` uses an N^2 calculation to generate the lists of neighbors that are within cutoff+skin.   
- The 'Nsqrd' was meant to differentiate between a neighbor list that uses a cell list in the calculation of which particles are neighbors (i.e. within cutoff+skin).   Whether consciously or not, I apparently mostly following the lammps naming here which refers to this style of neighbor list as "nsq" and the cell list approach as "bin" (i.e., binned). 
- Note, my initial testing of including a cell list was that it was slower over the range of system sizes we will be  targeting with NNPs.   It's not surprising that small systems would be more efficient using an N^2 neighbor generation, since it has less overhead, but the cross over to the cell list based approach being faster didn't seem to occur until about a million particles.  I was only testing on gpus, so this crossover may be much sooner if running on cpus.   I still plan to add in a `NeighborListCellList` (or some naming convention similar) that uses the CellList to calculate who are neighbors, but given the reduced efficiency in my initial tests, it got demoted in priority. 

I'll note, there is another routine available, named `PairList`.  This code generates a list of all pairs with no cutoff in the most efficient way for jax; the interaction cutoff doesn't come into play until we actually try to use it.  This is more efficient for really small LJ systems due to less overhead, but rapidly drops off as system size grows (not surprising since the actual pair calculation grows as N^2. )

Reply:
I'm naming the two existing functions:
- `NeighborListNsqrd` -- constructs a neighborlist to speed things up (but uses an N^2 scheme to calculate the neighbor list).  This requires a user to define a cutoff and skin.
- `PairListNsqrd` -- does not use a neighborlist, simply performs an N^2 calculation to see which particles are within the cutoff.  This was given the option to also perform with no cutoff (cutoff=none).  

the doc strings have been improved for both.

Reply:
I've also added more discussion in the example file.  I started making jupyter notebooks that will allow us to provide more discussion and background.